ci/recipe/build.sh

```diff
@@ -4,23 +4,34 @@ set -e
 set -x
 
 # extract Greengenes 13.8 reference files
+<<<<<<< HEAD
```

Comment:
Looks like some diff cruft was left behind, can you please fix this up?

Reply:
@thermokarst should be solved by now